source/includes/lifecycle-schedule-callout.rst

```diff
@@ -0,0 +1,6 @@
+.. important::
+
+   MongoDB maintains a support policy to ensure compatibility between MongoDB
+   Server and drivers for three years after the server version's end of
+   life (EOL) date. To learn more about the MongoDB release and EOL dates, see
```

Comment:
Q: is end of life usually written with hyphens?

Reply:
I think it can be either. I chose to use the version without hyphens since the linked page uses "end of life" rather than "end-of-life".